src/__tests__/events-compat.js

```diff
@@ -0,0 +1,27 @@
+import { h } from 'preact' // required by render
+import { fireEvent, render } from '..'
+import { forwardRef } from 'preact/compat' // required for this test to make sense
```

Comment:
```suggestion
import from 'preact/compat'
```

`preact/compat` is side-effectful, you don't need the unused `forwardRef` (or anything else)